packages/types/src/interfaces/contracts/runtime.ts

```diff
@@ -24,7 +24,7 @@ export const runtime: DefinitionsCall = {
             },
             {
               name: 'gasLimit',
-              type: 'u64'
+              type: 'Option<Weight>'
```

Comment:
This can't be - the runtime is supposed to be versioned, i.e. there should not be a breaking change without updating versions. (This is the same as above, this is breaking for users)

Reply:
IMO, `Option<Weight>` can be treated as `0u64`? and `0u64` can be treated as `None` ?

Reply:
There is a difference in SCALE encoding `Option<u64>` vs `u64`. The former would be 9 bytes in length, the later would be 8 bytes. So it breaks the structure decoding if wrong.